src/ctia/bundle/core.clj

```diff
@@ -109,14 +115,47 @@
              (all-pages entity-type external-ids auth-identity get-store))
       [])))
 
+(s/defn find-by-asset_refs :- {s/Str (s/pred map?)}
+  "Returns a map from asset_ref to entity of entity-type that has
+  entry `:asset_ref asset_ref`."
+  [asset_refs :- #{s/Str}
+   entity-type :- (s/enum :asset-properties :asset-mapping)
+   auth-identity :- auth/AuthIdentity
+   {{:keys [get-store]} :StoreService} :- FindByExternalIdsServices]
+  (if (empty? asset_refs)
+    {}
```

Comment:
I guess that's now it's already included in the first test of the loop, right?:
``` clojure
           entities (loop [asset_refs asset_refs
                          entities {}]
                     (if (empty? asset_refs)
                       entities
```

Reply:
I just left it in to skip a useless log.